http/http4s/src/main/scala/http/http4s.scala

```diff
@@ -17,17 +17,20 @@
 package freestyle
 package http
 
-import cats.{~>, Monad}
+import cats.{Applicative, Monad}
 import org.http4s.EntityEncoder
 
-import freestyle.implicits._
-
 object http4s {
 
   implicit def freeSEntityEncoder[F[_], G[_], A](
-      implicit NT: F ~> G,
+      implicit NT: FSHandler[F, G],
       G: Monad[G],
       EE: EntityEncoder[G[A]]): EntityEncoder[FreeS[F, A]] =
     EE.contramap((f: FreeS[F, A]) => f.exec[G])
 
+  implicit def freeSParEntityEncoder[F[_], G[_], A](
+      implicit NT: FSHandler[F, G],
+      G: Applicative[G],
+      EE: EntityEncoder[G[A]]): EntityEncoder[FreeS.Par[F, A]] =
+    EE.contramap((f: FreeS.Par[F, A]) => f.exec[G])
```

Comment:
:+1: This previously worked if `G` was a `Monad` as well through the implicit `FreeS.Par` to `FreeS` conversion, but I didn't think of the case where you interpret to "just" an `Applicative`.

So thanks for cleaning up after me!